### PR TITLE
SAP-S4HA10-setupguide-sle15.adoc sap-nw740-sle15-setupguide.adoc: obsoleted by simple-mount

### PR DESCRIPTION
**Pull request by lpinne**
Hi,
the two setup guides have been obsoleted by thesimple-mount setup.
However, the old concept still is supported for existing clusters.
The respective note might get published with the next update.
Regards,
Lars

### CODE REVIEW

File: adoc/sap-nw740-sle15-setupguide.adoc [hunk 1]
```diff
@@ -145,6 +148,19 @@ documentation and resources").
 
 For {saphana} system replication, follow the guides for the performance- or cost-optimized scenario.
 
+NOTE: The here described HA setup is based on cluster-controlled file systems for the
+SAP central services working directories.
+This setup with cluster-controlled file system resources has been obsoleted by the
+so-called simple-mount setup. The setup with cluster-controlled file systems still
+is supported for existing clusters.
```
Comment (dariavladykina) [inline adoc/sap-nw740-sle15-setupguide.adoc:155]:
```suggestion
so-called simple-mount setup. The setup with cluster-controlled file systems
is still supported for existing clusters.
```

File: adoc/sap-nw740-sle15-setupguide.adoc [hunk 2]
```diff
@@ -145,6 +148,19 @@ documentation and resources").
...
+For deploying new HA clusters please use the simple-mount setup, described in
```
Comment (dariavladykina) [inline adoc/sap-nw740-sle15-setupguide.adoc:156]:
```suggestion
For deploying new HA clusters, please use the simple-mount setup, described in
```

**Review by dariavladykina — COMMENTED** [review]
Hi, I've made some small suggestions for linguistic updates. Thanks!

**Comment by lpinne** [comment]
Hi Daria,
looks all good to me.
Thanks!
Lars

**Comment by dariavladykina** [comment]
Thank you @lpinne , I'll go ahead and commit them then.

**Review by fsundermeyer — COMMENTED** [review]
Looks good tto me! Any objections on merging?

**Comment by chabowski** [comment]
@lpinne : PR is merged, changes will be published with the next SAP repo builds.
One question: your comment means that we should keep the two docs because they are still valid for existing clusters, correct? We do not need to de-publish them? Thank you!

**Pull request merged**